dokka-subprojects/plugin-base/src/main/resources/dokka/styles/style.css

```diff
@@ -537,30 +513,6 @@ h1.cover {
     }
 }
 
-.main-content a:not([data-name]) {
-    padding-bottom: 2px;
-    border-bottom: 1px solid var(--border-color);
-    cursor: pointer;
-    text-decoration: none;
-    color: inherit;
-    font-size: inherit;
-    line-height: inherit;
-    transition: color .1s, border-color .1s;
-    overflow-wrap: break-word;
-    word-break: break-word;
-}
-
-.main-content a:hover {
-    border-bottom-color: unset;
-    color: inherit
-}
-
-a small {
-    font-size: 11px;
-    margin-top: -0.6em;
-    display: block;
-}
```

Comment:
I didn't find a situation where `small` element is presented. Especially inside the link element. 
So, I removed the style as obsolete.
There is also `Small` class in 'DocTag.kt', and I didn't find how the `Small` class is used.